playgame.py

```diff
@@ -59,17 +65,20 @@ def run_one_game(args):
 def prep_logger(log_dir, verbose, log_stderr, count):
     formatter = logging.Formatter('[%(asctime)s] %(levelname)8s --- %(message)s ' +
                                   '(%(filename)s:%(lineno)s)',datefmt='%Y-%m-%d %H:%M:%S')
+    
```

Comment:
nit: added whitespace, please remove

I'm less worried about it, but discipline from my former team is yelling in my head :)